jaxrs-api/src/main/java/jakarta/ws/rs/core/MediaType.java

```diff
@@ -82,15 +82,19 @@ public class MediaType {
     public static final MediaType APPLICATION_XHTML_XML_TYPE = new MediaType("application", "xhtml+xml");
     /**
      * A {@code String} constant representing {@value #APPLICATION_SVG_XML} media type.
+     * @deprecated since 4.0, use a custom string instead. Will be removed in a future release of this API.
      */
+    @Deprecated(forRemoval = true)
     public static final String APPLICATION_SVG_XML = "application/svg+xml";
     /**
      * A {@link MediaType} constant representing {@value #APPLICATION_SVG_XML} media type.
      */
     public static final MediaType APPLICATION_SVG_XML_TYPE = new MediaType("application", "svg+xml");
     /**
      * A {@code String} constant representing {@value #APPLICATION_JSON} media type.
+     * @deprecated since 4.0, use a custom {@code MediaType} instead. Will be removed in a future release of this API.
      */
+    @Deprecated(forRemoval = true)
```

Comment:
Why is this part of this PR? Title states scope is SVG.

Reply:
My assumption is this was intended for `APPLICATION_SVG_XML_TYPE` which is the variable above this one.

Reply:
Oops, sorry. James ist right, somehow the change slipped into the wrong lines.